backend/tests/v5/e2e/routes/teamspaces/projects/models/common/tickets.test.js

```diff
@@ -22,6 +22,7 @@ const ServiceHelper = require('../../../../../../helper/services');
 const { src, image } = require('../../../../../../helper/path');
 const { serialiseTicketTemplate } = require('../../../../../../../../src/v5/middleware/dataConverter/outputs/common/tickets.templates');
 const { queryOperators, specialQueryFields } = require('../../../../../../../../src/v5/schemas/tickets/tickets.filters');
+const { insertOne } = require('../../../../../../../../src/v5/handler/db');
```

Comment:
we already reference db in line 31. Also could you replace all the "./../../../../../../../src/v5/" references with `${src} as we do in line 28 for example ? (I know its from older code)